python/jupyter-notebooks/ohsome_green_region.ipynb

```diff
@@ -0,0 +1,876 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#read output dataset:\n",
+    "df_users = pd.read_pickle(\"./df_users.pkl\")\n",
+    "\n",
+    "val_u = {'region':['Vancouver', 'Halifax', 'Edmonton', 'Ottawa'], \n",
+    "'mean': [7.191082802547771, 2.878980891719745, 1.7197452229299364, 3.6815286624203822], \n",
+    "'median': [6.0, 2.0, 1.0, 3.0], 'min':[0.0, 0.0, 0.0, 0.0], 'max':[25.0, 13.0, 7.0, 13.0]}\n",
+    "val_users = pd.DataFrame(val_u)\n",
+    "\n",
+    "df_users.plot(kind='box', figsize=(12,10), fontsize=9)\n",
+    "display (val_users)\n",
+    "val_users.plot.barh(fontsize=9, figsize=(12,8))\n",
+    "\n",
+    "\n",
+    "#The table was created by searching for the mean, median, minimum and maximum values of each region.\n",
+    "#You can do it with numpy, like so:\n",
+    "#Ottawa, Ontario:\n",
+    "#Oarray = df['Ottawa']\n",
+    "#OttMean = np.mean(Oarray)\n",
+    "#OttMedian = np.median(Oarray)\n",
+    "#OttMin = np.min(Oarray)\n",
+    "#OttMax = np.max(Oarray)\n",
+    "#display(OttMean, OttMedian, OttMin, OttMax) -> to look at the values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bibliographic-vatican",
+   "metadata": {},
+   "source": [
+    "When looking at the minimum value of the user data output all the regions have the value 0, so during the time span looked at there were periods where there were no users of the request-tags at all.\n",
+    "Again, Edmonton hat the lowest values (mean, median & maximum) which might be yet another reason for the overall lower values in density of tree related polygon-tags.\n",
+    "\n",
+    "Halifax and Ottawa both have the same maximum value yet Ottawa is higher in mean as well as median. And the highest overall user-values are to be found for Vancouver.\n",
+    "\n",
+    "All regions but Edmonton have outliers which is in accord with the density values of Edmonton as there has been no visible mapping activity until 2011.\n",
+    "\n",
+    "**User activity Ranking:** \n",
+    "1. Vancouver\n",
+    "2. Ottawa\n",
+    "3. Halifax\n",
+    "4. Edmonton\n",
+    "\n",
+    "**Most stable user activity Ranking:**\n",
```

Comment:
What do you want to express with this ranking? So when I look for example at Edmonton: having a lower number of users is considered being better than having a generally higher number of users when the higher number has a bigger fluctuation? Hmm.. I am not sure if this ranking here is that meaningful.

Reply:
hmm yes, I mainly wanted to have the same amount of rankings for both requests but I wasn't so sure about this one either. I'll try to find something else, if not I'll probably take out the percentage ranking out too